MBBSEmu/HostProcess/MbbsHost.cs

```diff
@@ -1287,16 +1313,63 @@ private void DisableModule(string moduleId)
             _modules[moduleId].ModuleConfig.ModuleEnabled = false;
         }
 
+        private Timer SetupCleanupWarningTimer()
+        {
+            _cleanupWarningMinutesRemaining = CleanupWarningInitialMinutes;
+
+            var initialWarningTime = _cleanupTime + _cleanupGracePeriod - TimeSpan.FromMinutes(CleanupWarningInitialMinutes);
+            var dueTime = NowUntil(initialWarningTime);
+            var period = TimeSpan.FromMinutes(1);
+
+            Logger.Debug($"_cleanupTime Hours: {_cleanupTime.Hours}");
```

Comment:
We can probably remove these. While they're helpful developing this feature, I don't see us needing these printed every startup in debug.